app/shared-components/PromoModal/PromoModal.jsx

```diff
@@ -69,14 +69,14 @@ class PromoModal extends React.Component {
 		let url;
 		switch (product) {
 			case PLUS:
-				url = `https://checkout.${DOMAIN}.com/plus?utm_source=gbe&utm_campaign=${utm_campaign}`;
+				url = `https://www.${GHOSTERY_ROOT_DOMAIN}/products/plus?utm_source=gbe&utm_campaign=${utm_campaign}`;
 				break;
 			case PREMIUM:
-				url = `https://ghostery.com/thanks-for-downloading-midnight?utm_source=gbe&utm_campaign=${utm_campaign}`;
+				url = `https://www.${GHOSTERY_ROOT_DOMAIN}/midnight?utm_source=gbe&utm_campaign=${utm_campaign}`;
 				break;
 			case INSIGHTS:
 				sendMessage('ping', 'promo_modals_insights_upgrade_cta');
-				url = `https://checkout.${DOMAIN}.com/insights?utm_source=gbe&utm_campaign=${utm_campaign}`;
+				url = `https://checkout.${GHOSTERY_ROOT_DOMAIN}/insights?utm_source=gbe&utm_campaign=${utm_campaign}`;
```

Comment:
please use `CHECKOUT_BASE_URL` here

Reply:
Good catch! Changed to use `CHECKOUT_BASE_URL`

Reply:
in that case we should use `GHOSTERY_BASE_URL` for the others